sunpy/sun/sun.py

```diff
@@ -186,17 +186,15 @@ def true_obliquity_of_ecliptic(t='now'):
 
 def true_rightascension(t='now'):
     """Return the true right ascension."""
-    true_long = true_longitude(t)
-    ob = true_obliquity_of_ecliptic(t)
-    result = np.cos(ob) * np.sin(true_long)
-    result = result * u.deg
-    return Longitude(result)
+    y = np.cos(true_obliquity_of_ecliptic(t)) * np.sin(true_longitude(t))
+    x = np.cos(true_longitude(t))
+    true_ra = np.arctan2(y, x)
+    return Longitude(true_ra.to(u.hourangle))
 
 def true_declination(t='now'):
     """Return the true declination."""
-    result = np.cos(true_longitude(t))
-    result = result * u.deg
-    return Latitude(result)
+    result = np.arcsin(np.sin(true_obliquity_of_ecliptic(t))) * np.sin(apparent_longitude(t))
```

Comment:
why do you do the arcsin of a sin? shouldn't not be the same that the argument? Are you sure this is right? aren't you putting the parenthesis wrong? Look [my implementation](https://github.com/sunpy/sunpy/pull/1070/files#diff-1d9fb5cf2bfd8d46c960d4218bf8a3b0R374) based on "the :blue_book:"